### DISCUSSION TITLE
Process multiple files ?

### DISCUSSION THREAD

**Discussion by Cronos-lab** [xinntao/Real-ESRGAN#195] [Q&A]
Hello.

I would like to know if it is possible to use the tool with a list of files to process?
Now, I have to do it manually.

In any case, thanks for this tool, it's so beautiful !

Thank you

**Comment by Luna447** [comment]
I´d like to know that too.

**Reply by TechnoMasterBoy** [reply]
> I´d like to know that too.

Just follow what the readme tells you to do... 
realesrgan-ncnn-vulkan.exe -i input_folder -o outputfolder -n realesr-animevideov3 -s 4
Replace the "input_folder" with whatever your folder is called, including it's location. If it has spaces, put it in quotation marks. Same with output folder.

**Reply by Luna447** [reply]
Thanks man. I must have overlooked that. Have a nice day :)

**Reply by AssasinatorCzar** [reply]
did that work???
